db/warehouse/migrate/20230301172341_add_custom_assessments.rb

```diff
@@ -0,0 +1,82 @@
+class AddCustomAssessments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :CustomAssessments do |t|
+      t.string :EnrollmentID, null: false
+      t.string :PersonalID, null: false
+      t.string :UserID, limit: 32, null: false
+      t.date :AssessmentDate, null: false
+      t.integer :DataCollectionStage, null: false, comment: 'One of the HMIS 5.03.1, or 99 for local use'
+      t.integer :data_source_id
+
+      t.datetime :DateCreated, null: false
+      t.datetime :DateUpdated, null: false
+      t.datetime :DateDeleted
+    end
```

Comment:
Should this have an `AssessmentID`?

Reply:
Yeah, probably a `CustomAssessmentID` that's a UUID.  Good catch.

Reply:
Alright, added. This is ready for review now

Reply:
I'm probably blind, but I'm not  seeing it.

Reply:
🤦‍♀️  I had added them to my database schema, but not to this migration. Added!